src/app/containers/MostRead/index.jsx

```diff
@@ -1,12 +1,39 @@
 import React, { useContext } from 'react';
 import { oneOf, string, elementType, bool } from 'prop-types';
+import styled from 'styled-components';
+import SectionLabel from '@bbc/psammead-section-label';
 import { RequestContext } from '#contexts/RequestContext';
 import { ServiceContext } from '#contexts/ServiceContext';
 import useToggle from '#hooks/useToggle';
 import Canonical from './Canonical';
 import mostReadShape from './utilities/mostReadShape';
 import { getMostReadEndpoint } from '#lib/utilities/getMostReadUrls';
 
+export const MostReadSection = styled.section.attrs({
```

Comment:
Since these are separate exports, can `MostReadSection` & `MostReadSectionLabel` be in a separate file e.g. `src/app/containers/MostRead/section.jsx`.